helpdesk_mgmt_project/models/helpdesk_ticket.py

```diff
@@ -14,6 +15,15 @@ class HelpdeskTicket(models.Model):
         store=True,
     )
 
+    # Override field
+    company_id = fields.Many2one(
+        default=lambda self: self.env["project.project"]
```

Comment:
Hi,
Thanks for the work
prefer to define a method before the filed declaration and call the method